### PR TITLE
bsnes: Add version 115

### PR DESCRIPTION
**Pull request by samiksome92**
Add bsnes manifest.

### CODE REVIEW

**Comment by UnicorNora** [comment]
This package seems fine to me @Calinou

File: bucket/bsnes.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    ],
+    "persist": "settings.bml",
+    "checkver": {
+        "github": "https://github.com/bsnes-emu/bsnes",
+        "regex": "v([\\d.]+) "
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/bsnes-emu/bsnes/releases/download/v115/bsnes_v$version-windows.zip"
```
Comment (beyondmeat) [inline bucket/bsnes.json:29]: v115 needs to be v$version

**Comment by beyondmeat** [comment]
@samiksome92 still needs work. the autoupdate url is missing a $version replacement.
Title of PR should be "bnes: Add version 115"

**Comment by samiksome92** [comment]
@beyondmeat Fixed autoupdate url.

**Comment by beyondmeat** [comment]
superceeded by #583

**Comment by Calinou** [comment]
Superseded by #583. Thanks for the contribution nonetheless!

**Pull request closed without merging**